extensions/azuremonitor/src/ui-references.ts

```diff
@@ -0,0 +1,9 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Not sure why this file is named `ui-references`, that doesn't really indicate anything about what it is